common/numerated/src/tree.rs

```diff
@@ -161,20 +159,23 @@ impl<T: Numerated> IntervalsTree<T> {
         false
     }
 
-    /// The same as [`Self::contains`], but returns [`I::Error`] if `try_into` [Interval] fails.
-    pub fn try_contains<I: TryInto<Interval<T>>>(&self, interval: I) -> Result<bool, I::Error> {
-        let interval: Interval<T> = interval.try_into()?;
+    /// The same as [`Self::contains`], but returns [`I::Error`] if `try_into` [IntervalIterator] fails.
+    pub fn try_contains<I: TryInto<IntervalIterator<T>>>(
+        &self,
+        interval: I,
+    ) -> Result<bool, I::Error> {
+        let interval: IntervalIterator<T> = interval.try_into()?;
         Ok(self.contains(interval))
     }
 
     /// Insert interval into tree.
```

Comment:
It's maybe good to describe interval merging and splitting in `insert` and `remove` functions becasue iterators behavior becomes unobvious